service/versionbundle/version_bundle.go

```diff
@@ -45,3 +45,9 @@ func VersionBundle() versionbundle.Bundle {
 		Version: project.BundleVersion(),
 	}
 }
+
+func NewSlice() []versionbundle.Bundle {
```

Comment:
I find this confusing and would rather go with a slice literal in `main.go`. We will change the config type eventually in the future anyway because there should actually only ever be one version bundle, and not multiple once, right?

Reply:
> I find this confusing and would rather go with a slice literal in `main.go`.

Then I would have to alias one of `versionbundle` imports.

> We will change the config type eventually in the future anyway because there should actually only ever be one version bundle, and not multiple once, right?

Yes.

Reply:
I'm merging but we can still discuss this.